### DISCUSSION TITLE
How to change tissue model from perfusion limited to permeability limited model in PK-Sim

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1251] [How-Tos]
Dear Community,
I am working on a PBPK model in which I have to include diffusional clearance of the molecule across various tissues. For this I have to change the tissue model from being perfusion limited to permeability limited model. Where can I find the option to change the tissue model and how to specify the diffusional clearance in that?

Thank you in advance :)

**Comment by StephanSchaller** [comment]
Dear @Gautamvj , by default, the model contains both, perfusion and diffusion (to intracellular space). If you need a diffusive barrier for extravasation, you can switch to the "large molecule" model right at the start when creating a simulation:

![image](https://user-images.githubusercontent.com/25477643/178725208-5073f427-5a83-4e05-8170-941dc2bec378.png)

**Reply by Gautamvj** [reply]
Dear Stephan,

The drug molecule which I am working with is a transporter substrate and in literature along with the transporter kinetics there is also "diffusional clearance" reported in the units "microlitre/ml/million cells".  I specifically want to know where I can input this diffusional clearance. Also connected to this question, I also need to change the tissue-plasma partition coefficient (Kp) values for the tissues based on data obtained from tissue distribution studies. How can I change the Kp values for different tissues? I tried changing them in the simulation parameters but they are coming back to default.

Thank you.
Thank you.

**Comment by StephanSchaller** [comment]
Okay, well, we are working on this: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1489
Maybe we should add "microlitre/ml/million cells" as a dimension

Also you could use the total hepatic CL and calculate the half-life from your assay and enter it here:
![image](https://user-images.githubusercontent.com/25477643/179087809-fb2aa0a4-91c8-413a-8ba8-bef17da598fd.png)

**Reply by Gautamvj** [reply]
Can you also elaborate on this? - "Also connected to this question, I also need to change the tissue-plasma partition coefficient (Kp) values for the tissues based on data obtained from tissue distribution studies. How can I change the Kp values for different tissues? I tried changing them in the simulation parameters but they are coming back to default". That will be a great help.

Thanks a tonn!!!!
